libguard/guard_exception.hpp

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include <exception>
+#include <sstream>
+#include <string>
+
+namespace openpower
+{
+namespace guard
+{
+namespace exception
+{
+// TODO:Issue #3, to avoid multiple exception classes
+class GuardException : public std::exception
```

Comment:
Exception list provided in the exported interfaces need to revisit , not seeing bunch of exception related file operations here in the futnction hedare definition

Reply:
We are handling file open, seek, read and write error. Can you please share one is missed ?
Mostly file exceptions are thrown in file operations i.e. guard_file.cpp

Reply:
These guard fle  helper API's are consumed  exported interfaces right, You should look that also.

Reply:
Added the changes. Done